inc/util.h

```diff
@@ -78,19 +83,6 @@ int64_t parseIso8601Time(const char *str);
 
 uint64_t parseUnsignedInt(const char *str);
 
-// base64 encode bytes.  The output buffer must have at least
-// ((4 * (inLen + 1)) / 3) bytes in it.  Returns the number of bytes written
-// to [out].
-int base64Encode(const unsigned char *in, int inLen, char *out);
```

Comment:
yeah, that's caught me before, too! I think it does that with diffs larger than some threshold.